Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/modules/keysubmission/AnalyticsKeySubmissionCollector.kt

```diff
@@ -4,9 +4,11 @@ import de.rki.coronawarnapp.datadonation.analytics.common.calculateDaysSinceMost
 import de.rki.coronawarnapp.datadonation.analytics.common.toMetadataRiskLevel
```

Comment:
Please do not change anything about key submission. I'm doing this in a different PR